TUW_Radar_Image_Compositing/Python/gee_uc1_temp.py

```diff
@@ -0,0 +1,51 @@
+import openeo
+from openeo.internal.graph_building import PGNode
+import logging
+import time
+import json
+logging.basicConfig(level=logging.INFO)
+
+GEE_DRIVER_URL = "https://earthengine.openeo.org/v1.0"
+
+user = "group1"
+password = "test123"
+
+# Connect to backend via basic authentication
+con = openeo.connect(GEE_DRIVER_URL)
+con.authenticate_basic(user, password)
+
+datacube = con.load_collection("COPERNICUS/S1_GRD",
+                               spatial_extent={"west": 16.06, "south": 48.06, "east": 16.65, "north": 48.35, "crs": "EPSG:4326"},
+                               temporal_extent=["2017-03-01", "2017-06-01"],
+                               bands=["VV"])
+march = datacube.filter_temporal("2017-03-01", "2017-04-01")
+april = datacube.filter_temporal("2017-04-01", "2017-05-01")
+may = datacube.filter_temporal("2017-05-01", "2017-06-01")
+
+mean_march = march.mean_time(dimension="t")
```

Comment:
A general Python client question regarding `march.mean_time(dimension="t")` - why is there a `time` in the process name if you have to specify the dimension name anyway?

Reply:
You just have to set it if it is not "temporal" at the moment, but this is as far was I know, just for now. I think it is planned that the client will automatically set the dimensions according to the backend.

Reply:
This is now solved by the Python client (storing and reading the time dimension from the loaded collection) 
So that the "dimension" parameter is deprecated.

Reply:
since Open-EO/openeo-python-client#116 it's even invalid  (not just deprecated) to specify the dimension name

Reply:
I know, it was just necessary back then,  this is already fixed in my next commit (see https://github.com/Open-EO/openeo-usecases/pull/4/commits/924b392c1715bcad28b58ee7e774a4b79833c428)

Reply:
Oh sry I just noticed,  you just clarified the commend above